documents/customer.md

```diff
@@ -157,6 +157,248 @@ Razorpay::Customer.fetch(customerId)
+    "ifsc_code": "UTIB0000194",
+    "account_number": "916010082985661",
+    "beneficiary_name": "Pratheek",
+    "beneficiary_address1": "address 1",
+    "beneficiary_address2": "address 2",
+    "beneficiary_address3": "address 3",
+    "beneficiary_address4": "address 4",
+    "beneficiary_email": "random@email.com",
+    "beneficiary_mobile": "8762489310",
+    "beneficiary_city": "Bangalore",
+    "beneficiary_state": "KA",
+    "beneficiary_country": "IN"
+})
+```
+
+**Parameters:**
+
+| Name          | Type        | Description                                 |
+|---------------|-------------|---------------------------------------------|
+| customerId*  | string      | Unique identifier of the customer. |
+| account_number | string      | Customer's bank account number. For example, `0002020000304030434`. |
+| beneficiary_name | string  | The name of the beneficiary associated with the bank account.  |
+| beneficiary_address1 | string      | The virtual payment address.  |
+| beneficiary_email  | string      | Email address of the beneficiary. For example, `gaurav.kumar@example.com`. |
+| beneficiary_mobile | integer      | Mobile number of the beneficiary.  |
+| beneficiary_city  | string      | The name of the city of the beneficiary.  |
+| beneficiary_state | string      | The state of the beneficiary.  |
+| beneficiary_country | string      | The country of the beneficiary.  |
+| beneficiary_pin   | interger    | The pin code of the beneficiary's address.  |
+| ifsc_code          | string      | The IFSC code of the bank branch associated with the account.  |
+
+**Response:**
+```json
+{
+    "id": "ba_LSZht1Cm7xFTwF",
+    "entity": "bank_account",
+    "ifsc": "ICIC0001207",
+    "bank_name": "ICICI Bank",
+    "name": "Gaurav Kumar",
+    "notes": [],
+    "account_number": "XXXXXXXXXXXXXXX0434"
+}
+```
+
+-------------------------------------------------------------------------------------------------------
+
+### Delete Bank Account of Customer
+
+```rb
+customerId = "cust_N5mywh91sXB69O"
+
+bankAccountId = "ba_N6aM8uo64IzxHu"
+
+Razorpay::Customer.delete_bank_account(customerId, bankAccountId)
+```
+
+**Parameters:**
+
+| Name          | Type      | Description                                                       |
+|---------------|-----------|-------------------------------------------------------------------|
+| customerId*   | string    |  Customer id of the customer whose bank account is to be deleted. |
+| bankAccountId | string    | The bank_id that needs to be deleted.                             |
+
+**Response:**
+```json
+{
+    "id": "ba_Evg09Ll05SIPSD",
+    "ifsc": "ICIC0001207",
+    "bank_name": "ICICI Bank",
+    "name": "Test R4zorpay",
+    "account_number": "XXXXXXXXXXXXXXX0434",
+    "status": "deleted"
+}
+```
+
+-------------------------------------------------------------------------------------------------------
+
+### Eligibility Check API
+
+```rb
+Razorpay::Customer.requestEligibilityCheck({
```

Comment:
please update the function name here

Reply:
updated

https://github.com/razorpay/razorpay-ruby/blob/981d25ccff2474c3b1cbed16657d6a45606f7d35/documents/customer.md?plain=1#L246